doc/Language/objects.pod6

```diff
@@ -19,30 +19,28 @@ object. That includes literals, types (type objects), code and containers.
 
 =head1 Using Objects
 
-You can use objects by calling methods on them. To call a method on an
-expression, add a dot, followed by the method name:
+To call a method on an object, add a dot, followed by the method name:
 
 =for code :allow<B L>
 L<say> "abc"B<.L<uc>>;
 # ABC
 
 This calls the L<C<uc>|uc> method on C<"abc">, which is an object of type
-L<C<Str>|Str>. If you need to supply arguments to the method, add
-parentheses after the method name with the arguments inside the parentheses:
+L<C<Str>|Str>. To supply arguments to a method, add parentheses after the
+method, with arguments inside parentheses:
 
 =for code :allow<B L>
 my $formatted-text = "Fourscore and seven years ago...".L<indent>B<(8)>;
 
 C<$formatted-text> now contains the above text, but indented 8 spaces.
 
-Multiple arguments simply require a comma between them:
+Separate multiple arguments with commas:
 
 =for code :allow<B L>
 my @words = "Abe", "Lincoln";
 @words.L<push>("said"B<,> $formatted-text.L<comb>(L</\w+/|/language/regexes>));
 
-Another method call syntax separates the method name and the argument list
-with a colon:
+Alternately, separate the method and the argument list with a colon:
```

Comment:
Would be nice to have this the same as above, making it start with “Multiple arguments…”.